web/src/stories/CodeSnippet.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import CodeSnippet from "../components/shared/CodeSnippet";
+
+export default {
+  title: "Example/CodeSnippet",
+  component: CodeSnippet,
+} as ComponentMeta<typeof CodeSnippet>;
+
+const Template: ComponentStory<typeof CodeSnippet> = () => (
+  <CodeSnippet
+    language="bash"
```

Comment:
CodeSnippet and most likely some of the existing components will need rework so that the props will allow storybook to control.

Reply:
Depending on how much it is used we could make a new one with a different interface and swap out the old ones one at a time before deleting the old code. It's extra steps but I think it's worth it if it's really tangled up with other components.